AUTHORS

```diff
@@ -0,0 +1,7 @@
+# This is the list of func authors for copyright purposes.
+#
+# This does not necessarily list everyone who has contributed code, since in
+# some cases, their employer may be the copyright holder.  To see the full list
+# of contributors, see the revision history in source control.
+Red Hat, Inc.
+
```

Comment:
Is the list of authors intentionally empty?

Reply:
It include "Red Hat, Inc." on line 6. All contributors to date have done so as an employee of Red Hat.